docs/devsguide/new_dev_guide.rst

```diff
@@ -191,23 +103,66 @@ Contributing:
+And you don't just have to write code. You can help out by writing 
+documentation, tests, or even by giving feedback about this work. 
+(And yes, that includes giving feedback about the contribution guidelines.)
+
+Thank you for contributing!
+
+This disclaimer was originally written by `Adrienne Lowe 
+<https://github.com/adriennefriend/imposter-syndrome-disclaimer/blob/master/README.md>`_ 
+for a `PyCon talk <https://www.youtube.com/watch?v=6Uj746j9Heo>`_, and was adapted for
+PyNE's Guide for New Developers.
+
 ----------------
 Getting Practice
 ----------------
 Novices to open-source projects can get still contribute to PyNE.  
-To do so, go to PyNE’s `GitHub Page <https://github.com/pyne/pyne>`__ and, 
-on the right-hand side of the page, select Issues. Once on this page, select 
-the “low hanging pinoli” label to display more issues with the same tag.
-Pinoli is the Italian word for the Pine Nut, and this marker is the 
+To do so, go to PyNE’s `GitHub Page <https://github.com/pyne/pyne/issues>`__. Once
+on this page, select the “low hanging pinoli” label to display more issues with the
+same tag. Pinoli is the Italian word for the Pine Nut, and this marker is the
 first place New Developers should look to contribute.
 
----------------------------------
-Adding and Updating Documentation 
----------------------------------
-To contribute, you can edit the text file in any program that allows you to edit 
-text (Vim, TextEdit, Nano, etc.) and doesn’t invisibly add characters to the 
-file (like Word). The only important part is to write the file in a manner that’s 
-considered reStructuredText (check out http://sphinx-doc.org/rest.html). Then, 
-Sphinx will do everything else under the hood as described `here 
-<http://pyne.io/devsguide/website.html>`__. Finally, commit these changes to 
-your forked version and submit a pull request.
+---------------------
+Making a Pull Request
+---------------------
+Before you make the pull request (PR), make sure that you include a change to the 
+CHANGELOG.rst file under the appropriate heading. Your contribution can be as simple 
+as a line of text, but it should incorporate the PR's number in parenthesis at the end. 
+If there is a line already in the changelog that describes a similar action (e.g., 
+adding a publication) to your own, you can add the number of your PR after the existing line.
+
+The PR should contain a description of the changes, appropriate labels, projects,
+and a reference to the issue that led to it, which you can do by inserting
+‘#issue_number’ to the description (i.e., adding #161 to your PR description).
+These elements will help other contributors discover and review your work. Adding a
+reference to the issue the pull request will allow people to see the issue inspired
+it alongside any conversation about the issue.
+
+----------------------
+Changing Documentation
+----------------------
+To contribute, you can edit the text file in any program that allows you to edit
```

Comment:
Presumably one needs an editor for any of the above contributions, as well, right?  Why mention it here?  Perhaps because we imagine documentation contributions as lower hanging fruit and expect folks to enter here?

Reply:
This existed before @nsryan2's update, 

I believe the emphasis was more on the no MSWord than on the editor.